### PR TITLE
Grammar/Rewording

### PR DESCRIPTION
**Pull request by lewislarsen**
I've just rewritten a bit of this page, I found it earlier and it definitely needed some touch ups. I don't believe I've missed anything!

### CODE REVIEW

File: 1.8/administration/user-titles.md [hunk 1]
```diff
@@ -6,14 +6,17 @@ categories: [administration]
 
 ## About User Titles
 
-User titles are text displayed under the username in posts, threads, profile and member list.
-Each group has it's own default user title which is shown if a user doesn't use a custom one.
+User titles is text that is displayed under the username in posts, threads, profile and on the member list.
```
Comment (euantorano) [inline 1.8/administration/user-titles.md:9]: `User titles are text` sounds more grammatically correct to me.

File: 1.8/administration/user-titles.md [hunk 2]
```diff
@@ -6,14 +6,17 @@ categories: [administration]
...
+Each group has it's own default user title which is shown if a user doesn't use a custom user title.
 
 ## How to set Default User Titles
 
 You can set them to be default for one group or to be changed with number of posts a user has made.
 
-#### Set User Titles According the Group
+
+
```
Comment (euantorano) [inline 1.8/administration/user-titles.md:17]: What's with these extra blank lines?

**Review by euantorano — CHANGES_REQUESTED** [review]
Looks good, just some slight changes :smile:

**Comment by Ben-MyBB** [comment]
@LewisHolt any update on the changes?

File: 1.8/administration/user-titles.md [hunk 3]
```diff
@@ -6,14 +6,17 @@ categories: [administration]
...
+
+#### Set User Titles According to the Group
```
Comment (WildcardSearch) [inline 1.8/administration/user-titles.md:19]: This should be in **Title Case**

File: 1.8/administration/user-titles.md [hunk 4]
```diff
@@ -25,37 +28,37 @@ The "Registered" group by default has this option.
 
 If you want to use that option, leave Default User Title field in group settings blank.
 
-You can specify and edit number of posts, names or add new Custom User Titles at `Admin Control Panel -> Users & Groups -> User Titles`.
+You can specify the user title name, minimum number of posts, number of stars and the amount of stars at `Admin Control Panel -> Users & Groups -> User Titles`.
```
Comment (WildcardSearch) [inline 1.8/administration/user-titles.md:31]: Forgot the Oxford comma:

`number of stars, and the amount of stars`

File: 1.8/administration/user-titles.md [hunk 5]
```diff
@@ -25,37 +28,37 @@ The "Registered" group by default has this option.
...
 
 ##### Title to Assign field
 
-Here write text for Custom User Title
+Here you can write the text that you wish to be shown as the custom User Title. 
 
 ##### Minimum Posts
 
-Here write minimum number of posts, requied before user got assigned this Custom User Title.
+Here you can write the minimum number of posts required before the user will be assigned a Custom User Title.
```
Comment (WildcardSearch) [inline 1.8/administration/user-titles.md:41]: I'm thinking "write" should be "enter" or "set".

**Review by WildcardSearch — CHANGES_REQUESTED** [review]
Good work, just a few nits.

**Pull request closed without merging**